requirements/base.txt

```diff
@@ -9,36 +9,39 @@ backports.os==0.1.1       # via fs
 certifi==2019.9.11        # via requests
 chardet==3.0.4            # via requests
 django-model-utils==3.2.0
-django-waffle==0.17.0     # via edx-django-utils, edx-drf-extensions
-django==1.11.24
+django-waffle==0.18.0     # via edx-django-utils, edx-drf-extensions
+django==1.11.26 ; python_version < "3"
```

Comment:
This isn't correct, so we may need to give up on the upper constraint on Django in `base.in` and just leave it as `Django>=1.11`; I don't think there's a good way to get `pip-compile` to generate what we want here with an environment marker involved (there's an [open issue](https://github.com/jazzband/pip-tools/issues/518) for this).  I've updated the [BOM-1009](https://openedx.atlassian.net/browse/BOM-1009) description accordingly.